hawkbit-mgmt-resource/src/test/java/org/eclipse/hawkbit/mgmt/rest/resource/MgmtRolloutResourceTest.java

```diff
@@ -117,8 +119,44 @@ public void missingTargetFilterQueryInRollout() throws Exception {
     @Test
     @Description("Testing that rollout can be created")
     public void createRollout() throws Exception {
+        targetManagement.createTargets(testdataFactory.generateTargets(20, "target", "rollout"));
+
         final DistributionSet dsA = testdataFactory.createDistributionSet("");
-        postRollout("rollout1", 10, dsA.getId(), "name==target1");
+        postRollout("rollout1", 10, dsA.getId(), "id==target*");
+    }
+
+    @Test
+    @Description("Testing that rollout can be created with groups")
+    public void createRolloutWithGroupsDefinitions() throws Exception {
```

Comment:
Added #createRolloutWithIllegalPercentages() which tests 0 and 101 as illegal pecentages, which result in a constraint violation.